Pickers/AbstractActionSheetPicker.m

```diff
@@ -218,7 +254,7 @@ - (void)showActionSheetPicker {
 
 #pragma clang diagnostic push
 #pragma ide diagnostic ignored "UnavailableInDeploymentTarget"
-    if ([UIViewController instancesRespondToSelector:@selector(edgesForExtendedLayout)]) {
```

Comment:
If I understand you right, that is not necessary to remove this line right now, right?
If so, can your put back this line.
I believe it's needed to save compatibility with iOS < 7.0